mkdocs.yml

```diff
@@ -116,15 +120,17 @@ markdown_extensions:
       nested_indent: 2
       truly_sane: true
   - toc:
-        permalink: "#"
-        toc_depth: 3
+      permalink: "#"
+      toc_depth: 3
   - pymdownx.superfences:
       custom_fences:
         - name: mermaid
           class: mermaid
           format: !!python/name:pymdownx.superfences.fence_code_format
 
 extra:
+  k8s_version: 1.26.1
+  k0s_version: 1.26.1+k0s.0
```

Comment:
Hmm, isn't this bit more "dynamic" than this?

This is essentially a tag we're building right? Except for the main branch

Reply:
changed it to actually get the version from makefile variables